aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -253,6 +256,34 @@ func resourceAviatrixTransitGateway() *schema.Resource {
 				Computed:    true,
 				Description: "Private IP address of HA transit gateway.",
 			},
+			"bgp_polling_time": {
+				Type:         schema.TypeInt,
```

Comment:
Change type to String instead of Int. I think we only use TypeInt for cloud_type.